### PR TITLE
Fixed Lich Sacrifice Tooltip

### PR DESCRIPTION
**Pull request by Haganeko**
Remade lich sacrifice tooltip to properly reflect how the skill works in OAA

### CODE REVIEW

File: game/resource/English/ability/units/tooltip_lich_dark_ritual.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+//=============================================================================
+// LICH SACRIFICE
+//=============================================================================
+	"DOTA_Tooltip_ability_lich_dark_ritual"							"Sacrifice"
+	"DOTA_Tooltip_ability_lich_dark_ritual_Description"					"Sacrifices a creep and converts its current hit points into mana for Lich.\nCan be used on friendly or neutral creeps."
```
Comment (Chronophylos) [inline game/resource/English/ability/units/tooltip_lich_dark_ritual.txt]: We changed vanilla stuff right? We should color our changes. 
You could use some html
```html
Sacrifices a creep and converts its current hit points into mana for Lich.\nCan be used on friendly <font color=orange>or neutral creeps</font>.
```

Comment (chrisinajar) [inline game/resource/English/ability/units/tooltip_lich_dark_ritual.txt]: I love this idea.

Comment (Trildar) [inline game/resource/English/ability/units/tooltip_lich_dark_ritual.txt]: Should probably apply this to Call of Nature and Demonic Conversion as well.

Comment (chrisinajar) [inline game/resource/English/ability/units/tooltip_lich_dark_ritual.txt]: Yeah, we can do that as a separate unit of work and get all of them at once. I'll merge this now.

**Pull request merged**